.github/workflows/hotfix.yml

```diff
@@ -0,0 +1,40 @@
+name: "Hotfix"
+on:
+  pull_request:
+    branches:
+      - master
+    types: [closed]
+
+
+jobs:
+  build:
+    if: github.event.action == 'closed' && github.event.pull_request.merged == true && contains(github.event.pull_request.head.ref, 'hotfix/')
+
+    runs-on: ubuntu-16.04
+    container: rappdw/docker-java-python
```

Comment:
You need to update this image

Reply:
@rafaelleinio update this image in another PR, so to avoid conflict, I will not apply the change.